services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall=CinemaHall(pk=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(
+        session_date: Optional[str] = None
```

Comment:
If you specify a default value of `None` for a parameter, you don't need to use the `Optional` type hint